### ISSUE TITLE
What’s the biggest personal change you've ever made?

### ISSUE THREAD

**Issue by sindresorhus** [passy/ama#32]

**Comment by passy** [comment]
Love the question and I actually have a pretty simple answer to this: I became disciplined.

I don't know the date when it happened but it was actually a pretty decisive change from one day to the other. Not that I would stop doing _all_ the stupid things you would do while you were young, but all of a sudden I had a guiding principle that I didn't have before. Some of the changes that this caused where me stopping to eat _any_ kind of fast food or sweets. No pizza, no ice cream, no biscuits. Till this day, I haven't made a single exception to this.

I also started to go to the gym. At the time where I made that decision I wasn't in the best shape which was probably the biggest trigger. I went three times a week. From doing no sports at all to doing something three times a week is already a good change, but it didn't take me long to ramp this up to going 5 - 6 times a week for at least 2 hours per session.

At this point, it is way easier for me to do something consistently always, every day than maintaining some elaborate set of rules when exceptions. My GitHub streak is another artifact of this. I'm not sure how healthy this particular thing actually is, and I'm certainly not running around telling people to do it, but it's something that works for me. If I wouldn't do it every day, chances are good that I wouldn't do it at all. The same is probably true for the gym. For this entire year, I've been to a gym every single day. With the exception of three days that I was on a ski trip, but since there was definitely no shortage of physical activity I was happy to adjust my daily goals for this one - and there've also been plenty of multiple gym sessions per day recently, so the 365/365 is still very much achievable.

There's a fine line between being disciplined and obsessed. I'm not sure which side I'm on and I can't apply it to everything - it has to be something I care about at least. But it's definitely been a very fundamental change and something I do not regret.
